metadata.json

```diff
@@ -4,5 +4,5 @@
     "name": "Mute spotify ads",
     "version": 2,
     "url": "https://github.com/danigm/spotify-ad-blocker",
-    "shell-version": ["3.38"]
+    "shell-version": ["40"]
```

Comment:
Are you trying this extension in gnome-40? I've not tested in gnome-40 yet, so I'm not sure that this continues working there without any change.

Reply:
@danigm No, I didn't tested it. I am new to opensource. Can you tell me how to test it? Do it need to install the gnome 40 in VM  or some shortcut is there? Or can't we just contact the person @amitanilkulkarni who raised the issue to test it?

Reply:
It's possible to test with a virtual machine, using the https://os.gnome.org/. I've not tested yet there because I'm using ubuntu and gnome-40 is not coming to ubuntu for the next release, but at some point we can update the extension to work on gnome-40 and do a release for that version.

Reply:
I am also using Ubuntu but I am just trying to solve the issues in my favourite extension. I am new in opensource so can you suggest some projects I can contribute to.

Reply:
That's cool, working on extensions is a good way to start collaborate, and it's always good to try to collaborate in software that you use. If you want to find other projects to collaborate, GNOME has a newcomers page with more information: https://wiki.gnome.org/Newcomers/

In any case, for this PR, I would suggest you to split in two different PR, because the second commit could be merged now, but the gnome-40 could need more changes if it's not  working correctly.

Reply:
@danigm - For GNOME 40, the changes required are in settings UI only afaik. Since there's no settings UI for this one, adding 40.0 to shell-version (alongwith 3.38) might just be enough.

I could enable it in Extensions in Fedora 34 Beta by changing the shell-version to 40.0.

Reply:
Great than it can be merged